server/jetstream_cluster.go

```diff
@@ -3997,7 +3997,7 @@ func (js *jetStream) processClusterCreateConsumer(ca *consumerAssignment, state
 	var didCreate, isConfigUpdate, needsLocalResponse bool
 	if o == nil {
 		// Add in the consumer if needed.
-		if o, err = mset.addConsumerWithAssignment(ca.Config, ca.Name, ca, false); err == nil {
+		if o, err = mset.addConsumerWithAssignment(ca.Config, ca.Name, ca, false, ""); err == nil {
```

Comment:
Since we send CA why do we need extra arg here?

Reply:
as `action` does make sense only in context of given request, it didn't feel right to contain it as part of `CA`. Hence a separate parameter, extracted from Consumer Request.